src/data/string/ustring.h

```diff
@@ -75,72 +76,121 @@ uchar string_to_uchar(const char* p);
 uchar string_to_uchar(const std::string& s);
 std::string uchar_to_string(uchar uc);
 
+class fallback_base {
+public:
+  virtual ~fallback_base() {
+  }
+
+  virtual uchar fallback(const std::string& bytes,
+			 const char* hint) = 0;
```

Comment:
Hard tab indent(`\t`) is used here(and more 14 lines),
but pficommon's coding standard prefer to use soft tab(`\x20`) indent.
Please expand hard tab to soft tab.